flow/ApiVersions.cmake

```diff
@@ -1,27 +1,27 @@
 # API Versions
-set(FDB_AV_LATEST_VERSION                   "730")
-set(FDB_AV_LATEST_BINDINGS_VERSION          "720")
+set(FDB_AV_LATEST_VERSION                   "710300")
+set(FDB_AV_LATEST_BINDINGS_VERSION          "710200")
```

Comment:
As discussed, set FDB_AV_LATEST_BINDINGS_VERSION to 710300 too

Reply:
Oh, sorry, I missed that. Updated now.